sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundle.java

```diff
@@ -21,77 +21,34 @@
 
 import com.google.gson.Gson;
 import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Arrays;
-import java.util.Enumeration;
 import java.util.List;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipFile;
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.config.Configuration;
-import org.sonar.api.utils.log.Logger;
-import org.sonar.api.utils.log.Loggers;
 import org.sonar.plugin.typescript.SensorContextUtils;
 import org.sonar.plugin.typescript.SensorContextUtils.RuleToExecute;
 import org.sonar.plugin.typescript.TypeScriptPlugin;
 import org.sonar.plugin.typescript.TypeScriptRules;
 
 public class SonarTSCoreBundle implements ExecutableBundle {
 
-  private static final Logger LOG = Loggers.get(SonarTSCoreBundle.class);
   private static final int NODE_PROCESS_MEMORY = 2048;
 
   // relative location inside sonarts-core bundle
   private static final String BIN = "node_modules/tslint-sonarts/bin/";
   private final Configuration configuration;
 
-  private File deployDestination;
+  private final File sonartsRunnerExecutable;
+  private final File sonartsServerExecutable;
 
-  private String bundleLocation;
-  private File sonartsRunnerExecutable;
-  private File sonartsServerExecutable;
-
-  private SonarTSCoreBundle(String bundleLocation, File deployDestination, Configuration configuration) {
-    this.bundleLocation = bundleLocation;
-    this.deployDestination = deployDestination;
+  public SonarTSCoreBundle(File baseDir, Configuration configuration) {
```

Comment:
@saberduck can we rename `baseDir`? sonar-api uses this name for different

Reply:
@saberduck is it fixed?

Reply:
I forgot this one, fixed now.